src/client/pages/LoginApp/components/LoginForm.tsx

```diff
@@ -78,8 +75,12 @@ const SubmitButton = styled(Button)<{ disabled?: boolean }>`
 export const LoginForm: React.FC<LoginFormProps> = ({ onSubmit }) => {
   const textKeys = useTextKeys()
   const market = useMarket()
-  const ssnMaxLength = ssnLengthByMarket[market]
-  const ssnRegExp = ssnRegExpByMarket[market]
+
+  const currentLocale = useCurrentLocale()
```

Comment:
Should the return value of this hook to updated to `LocaleLabel`? Or Another hook that directly returns "currentLocaleData"?

Reply:
I think it probably should. But then my idea is to get rid of that hook altogether, using a context to store the current locale with all the locale specific data. So that's why I left it like this. It's kind of an in-between state, but I didn't want to do the whole big switch here, seeing that we have a lot of other things to do etc. 😅

Reply:
I would vote for a separate hook in that case :)